CONTRIBUTING.md

```diff
@@ -1,5 +1,45 @@
 # Contributing to Deephaven in VS Code
 
+## Development
+
+### Unit Testing
+
+TODO:
```

Comment:
TODO - fill in or write a ticket

Reply:
Yes, the prettierrc thing does apply to json, but that already works as expected. The issue here is that prettier incorrectly adds trailing commas to jsonc. It doesn't do that for json